### ISSUE TITLE
increase attack speed of defense towers, snake pits

### ISSUE THREAD

**Issue by andy5995** [ZetaGlest/zetaglest-data#142]
Anyone else think the attack speed of defense towers should be increased?

This would apply to things like

 * persian snake basket
 * tech defense tower
 * roman guard tower

and other similar units.

Right now, people seem to mostly just build beehives, but otherwise people are rarely building defense towers. Why bother maintaining units that are hardly ever used? I think it would be good to make an incentive for them to be built and played!

Imagine there are 4 archers in a guard tower.. If that upsets balance, we can look at changing some of their other attributes (cost, build time, etc).

**Comment by Jammyjamjamman** [comment]
I'm not too bothered about tech and (I think?) greece guard towers, since these are already pretty strong.

I made some significant changes to the persian defense units, especially the snake basket. My idea was: make the snake basket quick to build, so it can potentially be built on a battlefield. The penalty is I halved its strength.

I boosted tower romans tower because I decided romans was generally a bit weak compared to other factions, currently.
